### PR TITLE
Fix/version header

### LINKED ISSUE #54: Header sending `angular-instantsearch: undefined` as version
Thank's @mthuret for the report.

### PR DESCRIPTION
**Pull request by iam4x**
fixes #54

### CODE REVIEW

**Comment by iam4x** [comment]
Deploy preview ready!

Built with commit 2de2243b4132ee5122347f9a4c917e5d120d4a6d

https://deploy-preview-55--angular-instantsearch.netlify.com

File: src/version.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const VERSION = "1.0.0-beta.7";
```
Comment (Haroenv) [inline src/version.ts]: Will this be updated in the release script?

File: scripts/release.sh [hunk 2]
```diff
@@ -20,11 +20,12 @@ echo "What is the next version?"
 read next_version
 
 # replace package.json with next version
+sed -i.bak "s/${actual_version}/${next_version}/g" src/version.ts
```
Comment (iam4x) [inline scripts/release.sh]: @Haroenv Here 👍

Comment (Haroenv) [reply scripts/release.sh]: all right, glanced over it

**Pull request merged**